Update cleaning info for vanilla masters

z929669 opened the pull request:
Added latest xEditQuickAutoClean data for v1.6.1170 vanilla masters

LOOT 0.22.3 isn't correctly flagging all applicable plugins for cleaning for the latest game files for Steam v1.6.1170 (Anniversary Upgrade), and some of the current masterlist cleaning data is outdated in relation to the latest game files, so I'm attempting to remedy this. 
First time contributing, so I have only added updated cleaning info for the official game files and the CC base game files. My xEdit summaries are a bit different with the latest version of xEdit, so I'd like some guidance if I've included redundant info. See the attached archive for complete summary and evidence I'm using here.

Thanks

[Autoclean - SSE-1.6.1170.zip](https://github.com/loot/skyrimse/files/14752332/Autoclean.-.SSE-1.6.1170.zip)

z929669 committed:
Update cleaning info for vanilla masters

Added latest xEditQuickAutoClean data for v1.6.1170 vanilla masters

pStyl3 commented on masterlist.yaml:
```diff
@@ -898,6 +898,17 @@ plugins:
         udr: 93
         nav: 3
+      # v1.6.1170.0
+      - <<: *reqManualFix
```
While technically correct, for the _vanilla game files_ we manually adjust the reports to say `*quickClean` instead of `*reqManualFix` so this needs to be changed.

pStyl3 commented on masterlist.yaml:
```diff
@@ -898,6 +898,17 @@ plugins:
+        itm: 386
+        udr: 93
+        nav: 3
+      - <<: *reqManualFix
+        crc: 0xA14EED01
+        util: '[SSEEdit v4.1.5d](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        nav: 3
```
This second entry can be removed.

pStyl3 commented on masterlist.yaml:
```diff
@@ -949,6 +960,22 @@ plugins:
         udr: 82
         nav: 59
+      # v1.6.1170.0
+      - <<: *reqManualFix
```
This needs to be changed to `*quickClean`.

pStyl3 commented on masterlist.yaml:
```diff
@@ -949,6 +960,22 @@ plugins:
+        itm: 842
+        udr: 82
+        nav: 59
+      - <<: *reqManualFix
+        crc: 0xA62C1C10
+        util: '[SSEEdit v4.1.5d](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        itm: 7
+        nav: 59
+      - <<: *reqManualFix
+        crc: 0xA5AB2927
+        util: '[SSEEdit v4.1.5d](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        nav: 59
```
These can be removed.

z929669 replied:
When there are two passes performed in xEdit (as is uniquely the case for Dawnguard.esm it seems), shouldn't the itm/udr be summed for Pass 1 and Pass 2? So ...
```
      - <<: *reqManualFix
        crc: 0xD399544A
        util: '[SSEEdit v4.1.5d](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        itm: 842
        udr: 82
        nav: 59
      - <<: *reqManualFix
        crc: 0xA62C1C10
        util: '[SSEEdit v4.1.5d](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        itm: 7
        nav: 59
```
would be ...
```
      - <<: *reqManualFix
        crc: 0xD399544A
        util: '[SSEEdit v4.1.5d](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        itm: 849
        udr: 82
        nav: 59
```

pStyl3 replied:
I don't think that doing this is necessary. By adding only the first (unchanged) result from cleaning the file we already have accomplished that the user is informed about the plugin being dirty.

pStyl3 commented on masterlist.yaml:
```diff
@@ -990,6 +1017,17 @@ plugins:
         udr: 11
         nav: 5
+      # v1.6.1170.0
+      - <<: *reqManualFix
```
This needs to be changed to `*quickClean`.

pStyl3 commented on masterlist.yaml:
```diff
@@ -990,6 +1017,17 @@ plugins:
+        itm: 221
+        udr: 11
+        nav: 5
+      - <<: *reqManualFix
+        crc: 0x2913D707
+        util: '[SSEEdit v4.1.5d](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        nav: 5
```
These can be removed.

pStyl3 commented on masterlist.yaml:
```diff
@@ -1039,6 +1077,17 @@ plugins:
         udr: 8
         nav: 1
+      # v1.6.1170.0
+      - <<: *reqManualFix
```
This needs to be changed to `*quickClean`.

pStyl3 commented on masterlist.yaml:
```diff
@@ -1039,6 +1077,17 @@ plugins:
+        itm: 88
+        udr: 8
+        nav: 1
+      - <<: *reqManualFix
+        crc: 0x077954E6
+        util: '[SSEEdit v4.1.5d](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        nav: 1
```
This must be removed.

pStyl3 commented on masterlist.yaml:
```diff
@@ -1070,6 +1119,16 @@ plugins:
         itm: 3
         udr: 4
+    # v1.6.1170.0
+    dirty:
```
This second `dirty` tag must be removed.

pStyl3 commented on masterlist.yaml:
```diff
@@ -1070,6 +1119,16 @@ plugins:
+        util: '[SSEEdit v4.1.5d](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        itm: 3
+        udr: 4
+    clean:
+      - crc: 0x6B3BE132
+        util: 'SSEEdit v4.1.5d'
```
We don't add the `clean` info for dirty mods, so this should be removed.

pStyl3 commented on masterlist.yaml:
```diff
@@ -1081,11 +1140,24 @@ plugins:
         util: '[SSEEdit v4.0.4c](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 1
+    # v1.6.1170.0
+    dirty:
```
This second `dirty` tag must be removed.

pStyl3 replied:
This second `dirty` tag must be removed.

pStyl3 commented on masterlist.yaml:
```diff
@@ -1081,11 +1140,24 @@ plugins:
+        crc: 0x25BC177E
+        util: '[SSEEdit v4.1.5d](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        itm: 1
+    clean:
+      - crc: 0x518EFEB6
+        util: 'SSEEdit v4.1.5d'
```
We don't add the `clean` info for dirty mods, so this should be removed.

pStyl3 commented on masterlist.yaml:
```diff
@@ -1097,8 +1169,21 @@ plugins:
         util: '[SSEEdit v4.0.4c](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 3
+    # v1.6.1170.0
+    dirty:
```
This second `dirty` tag must be removed.

pStyl3 commented on masterlist.yaml:
```diff
@@ -1097,8 +1169,21 @@ plugins:
+        crc: 0xEBBC9DA9
+        util: '[SSEEdit v4.1.5d](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        itm: 3
+    clean:
+      - crc: 0x84B79395
+        util: 'SSEEdit v4.1.5d'
```
We don't add the `clean` info for dirty mods, so this should be removed.

pStyl3 commented on masterlist.yaml:
```diff
@@ -1097,8 +1169,21 @@ plugins:
+        util: 'SSEEdit v4.1.5d'
   # Resource Pack
   - name: '_ResourcePack.esl'
+    # v1.6.1170.0
+    clean:
+      - crc: 0x7898501C
+        util: 'SSEEdit v4.1.5d'
```
The cleaning info should be placed below the `group` entry.

pStyl3 reviewed (changes requested):
Thanks, there are a few things that need to be changed before we can merge the PR.

z929669 committed:
Update cleaning info for vanilla masters (#2)

Second update after verifying the expected patterns with the repo admin. All 79 core + base game CC + Anniversary Edition CC have been updated.

z929669 committed:
Update cleaning info for vanilla masters (pass 2)

Second update after verifying the expected patterns with the repo admin. All 79 core + base game CC + Anniversary Edition CC have been updated.

z929669 commented:
Not sure how the admins want to proceed, but I've made the requested changes and added updated info likewise for all the remaining v1.6.1170.0 vanilla masters. This has been done on a new commit (940a0fc3d042540275ef84c7ca19cf4a945052af) now that I better understand the requirements gleaned from this pull request.

z929669 committed:
Merge branch 'v0.21' of https://github.com/z929669/skyrimse into v0.21

z929669 commented:
Apologies, but I see my summary message for my last commit was 'bad' in that I included `#2` in the commit summary, which incorrectly linked the loot#2 issue to the commit, so I revised the summary message and merged into the latest commit (6cfe9d6c41aa392d899046e8ab42350d3e4316ba). Hopefully, I did this correctly.

pStyl3 commented on masterlist.yaml:
```diff
@@ -1081,11 +1115,20 @@ plugins:
       - crc: 0xEA917D0B
         util: 'SSEEdit v4.0.4c'
+    # v1.6.1170.0
+    clean:
```
This duplicate `clean` tag must be removed.

pStyl3 commented:
> Apologies, but I see my summary message for my last commit was 'bad' in that I included `#2` in the commit summary, which incorrectly linked the #2 issue to the commit, so I revised the summary message and merged into the latest commit ([6cfe9d6](https://github.com/loot/skyrimse/commit/6cfe9d6c41aa392d899046e8ab42350d3e4316ba)). Hopefully, I did this correctly.

This shouldn't be an issue. Once everything is ready to be merged, we usually squash merge the PR either way.

z929669 committed:
Removed redundant 'clean:' tag

z929669 commented:
Latest commit has the requested change
